### PR TITLE
Limit FPS when upscaling is disabled

### PR DESCRIPTION
**Pull request by AJenbo**

### CODE REVIEW

**Comment by glebm** [comment]
Why only when upscale is disabled?

**Comment by AJenbo** [comment]
@glebm because there is no vsync when outputting directly to the window sufrace meaning that we will output at 100% CPU
https://github.com/diasurgical/devilutionX/pull/461/files#diff-af7e08774858fbb4933bf5cf16fe0e66R1186

**Comment by glebm** [comment]
Let's check for `renderer` instead of `upscale`?

**Comment by AJenbo** [comment]
Any idea for how to fix this for SDL1?

**Comment by AJenbo** [comment]
@glebm on second thought it might not be as simple to use `renderer` as it isn't exposed at this part of the code.

**Comment by AJenbo** [comment]
@glebm I looked at SDL1.2 this morning and it doesn't appear to have a way to query the monitor refresh rate, so instead I think I will simply cap it at 60fps for that target.

File: SourceX/miniwin/misc.cpp [hunk 1]
```diff
@@ -190,6 +190,14 @@ bool SpawnWindow(LPCSTR lpWindowName, int nWidth, int nHeight)
 		if (SDL_RenderSetLogicalSize(renderer, nWidth, nHeight) <= -1) {
 			ErrSdl();
 		}
+#endif
+	} else {
+#ifdef USE_SDL1
+		refreshDelay = 16; // rougly 60hz
+#else
+		SDL_DisplayMode mode;
+		SDL_GetDisplayMode(0, 0, &mode);
```
Comment (AJenbo) [inline SourceX/miniwin/misc.cpp]: This query first monitor, technically there can be several but then we would also have to figure out what monitor we are running on, and the user may change that at any time. So given that `!upscal` is not seen as the primer target I think this as an acceptable solution.

**Pull request merged**